utils.py

```diff
@@ -100,12 +101,15 @@ def import_article_metadata(request, reader):
                         or parse_date(date_published))
                 article.stage = stage
                 article.doi = doi
+                sec_obj, created = submission_models.Section.objects.language(
```

Comment:
Should we check if `section is not None` first?

Reply:
We should probably check this and crap out the whole operation as section is a required field.